### PR TITLE
Improve Fragment Requirement Hints Translation

### PR DESCRIPTION
**Pull request by Griphen116**
Fragment 4 and fragment 10 - "Help from the queen carrier is required." might be better translated as "Cooperation from the queen carrier is required." or "Queen carrier must be willing to participate." 

Fragment 14 - "Satoshi Houjou must not be allowed to visit his sister at the clinic." might be better translated as "Satoshi Houjou's sister must refuse his visit at the clinic." 

Fragment 12 - "The research needs to have stagnated" might be better translated as "The research needs to have hit an obstacle." to be accurate or "The research needs to have hit the first snag." to match what the other instance of the lines translation.

### CODE REVIEW

**Comment by drojf** [comment]
There has been some discussion on discord here: https://discord.com/channels/384426173821616128/392489108875771906/1064196758214819860 about whether the translation for the fragment hints can be improved. 

This PR modifies the English translation of the hint for fragments 4, 10, 12, and 14 to be more accurate and/or less confusing.

**Comment by orian34** [comment]
F4 F10 Cooperation seems to be the better wording choice here.
F12 could go for "The research needs to have stagnated for weeks."

File: Data/fragmentdata.txt [hunk 1]
```diff
@@ -29,7 +29,7 @@
 	{
 		"Id":4,
 		"Title":"Research is going well",
-		"Description":"An L5 stage patient's dissection is required.\nHelp from the queen carrier is required.",
+		"Description":"An L5 stage patient's dissection is required.\nQueen carrier must be willing to participate.",
```
Comment (drojf) [inline Data/fragmentdata.txt:32]:
```suggestion
		"Description":"An L5 stage patient's dissection is required.\nQueen carrier must be cooperative.",
```

File: Data/fragmentdata.txt [hunk 2]
```diff
@@ -101,7 +101,7 @@
 	{
 		"Id":12,
 		"Title":"Requesting assistance from a queen",
-		"Description":"The research needs to have stagnated.\nInstructions for the kidnapping operation of the Minister's grandson are required.",
+		"Description":"The research needs to have hit the first snag.\nInstructions for the kidnapping operation of the Minister's grandson are required.",
```
Comment (drojf) [inline Data/fragmentdata.txt:104]:
```suggestion
		"Description":"The research needs to have stagnated for weeks.\nInstructions for the kidnapping operation of the Minister's grandson are required.",
```

File: Data/fragmentdata.txt [hunk 3]
```diff
@@ -83,7 +83,7 @@
 	{
 		"Id":10,
 		"Title":"At a quarry in Yagouchi",
-		"Description":"Help from the queen carrier is required.",
+		"Description":"Willingness to help from the queen carrier is required.",
```
Comment (drojf) [inline Data/fragmentdata.txt:86]:
```suggestion
		"Description":"Cooperation from the queen carrier is required.",
```

**Comment by drojf** [comment]
I've just added what orian wrote as suggestions so we can see it easier (I'm not vouching for/against it)

File: Data/fragmentdata.txt (hunk 1 above)
Comment (drojf) [reply Data/fragmentdata.txt:32]: Not sure if the wording I used is what orian intended here.

**Review by drojf — COMMENTED** [review]
I've added this review just to preview Orian's suggested changes (hope I did them correctly)

Edit: updated to match the wording used in the required fragments

File: Data/fragmentdata.txt (hunk 1 above)
Comment (orian34) [reply Data/fragmentdata.txt:32]: That one's the same as the one under, I meant "Cooperation from the queen carrier is required." Queen carrier isn't a name so it's weird to use it as such.

Comment (drojf) [reply Data/fragmentdata.txt:32]: ok, I changed it to the same as number 10, "Cooperation from the queen carrier is required."

**Comment by Griphen116** [comment]
@orian34 we made the wording to be close to what was used in the requirement fragments to avoid different translations for the same phrase in requirement hint and the required fragment. 
Rika says she is "willing to help" in the required fragment.
Irie says research has "hit the first snag" in the required fragment.

**Comment by orian34** [comment]
@Griphen116 It's a bit awkward, but if you think it's necessary, then I guess it's fine.

File: Data/fragmentdata.txt (hunk 1 above)
Comment (drojf) [reply Data/fragmentdata.txt:32]: For reference, the phrasing used in the fragment is "Rika is willing to help"

https://github.com/07th-mod/matsuribayashi/blob/89ed4b2dd1c425ebca24889979de4a3eba3f31de/Update/_kakera07.txt#L829

File: Data/fragmentdata.txt (hunk 2 above)
Comment (drojf) [reply Data/fragmentdata.txt:104]: For reference, the phrasing used in the fragment is "having its first snag"

https://github.com/07th-mod/matsuribayashi/blob/89ed4b2dd1c425ebca24889979de4a3eba3f31de/Update/_kakera18.txt#L267

File: Data/fragmentdata.txt (hunk 3 above)
Comment (drojf) [reply Data/fragmentdata.txt:86]: For reference, the phrasing used in the fragment is "Rika is willing to help"

https://github.com/07th-mod/matsuribayashi/blob/89ed4b2dd1c425ebca24889979de4a3eba3f31de/Update/_kakera07.txt#L829

Comment (drojf) [reply Data/fragmentdata.txt:86]: Actually I linked the wrong fragment, the actual wording in fragment 12 is "I'm willing to participate"

https://github.com/07th-mod/matsuribayashi/blob/714015600b14fb7270b5b9917b7e9a4a867bf098/Update/_kakera12.txt#L878

**Comment by drojf** [comment]
@Griphen116 

Shouldn't both fragment 4 and 10 use the same wording, "Queen carrier must be willing to participate." then? Currently fragment 10 uses a different wording "Willingness to help from the queen carrier is required." . Unless this is being done on purpose to make it less obvious that both fragments require fragment 12.

Also, should there be a "The" at the start, like "**The** Queen carrier must be willing to participate."?

**Comment by Griphen116** [comment]
Yes if they have the same fragment requirement, they should have the same wording in the hint.

**Review by drojf — CHANGES_REQUESTED** [review]
As discussed:
- Add "The" at start of "Queen carrier must be willing to participate"
- Make wording of fragment 4 and fragment 10 the same, as they both require fragment 12

Also I changed "hit the first snag" to "hit its first snag" to better match the fragment's wording, not sure if that is a good change.

**Pull request merged**